src/config.ts

```diff
@@ -25,6 +30,17 @@ export const bcParentGroup = (newName = ''): string | boolean => {
     } else {
         try {
             fs.writeFileSync('parentGroup.txt', newName);
+            try {
+                const list = await fetchGroups();
+                if (list?.length < 1) {
+                    channel.send(
```

Comment:
Move sending messages outside of this function, for e.g. there where you are calling this (server.ts). then you don't need to pass on the channel for this function, and this function does not have to be async, nor waited, and is easier to test. 
Pay also attention to that try inside of a try, returning true and catching true. Does it make sense?